supplier/tests/test_supplier_page.py

```diff
@@ -39,3 +40,41 @@ def test_supplier_check_false(non_supplier_mock):
 def test_supplier_check_true(saved_supplier0):
     assert saved_supplier0 in Supplier.objects.all()
     assert get_supplier(saved_supplier0) == saved_supplier0
+
+
+@pytest.mark.django_db()
+def test_supplier_deliveries(client, delivery_location0):
+    delivery_location0.add_delivery_location()
+    client.force_login(user=delivery_location0.user_name.supplier_account)
+    response = client.get('/suppliers/')
+    assert list(response.context['supplier_deliveries']) == [delivery_location0]
+
+
+@pytest.mark.django_db()
+def test_create_delivery(client, saved_supplier0):
+    client.force_login(user=saved_supplier0.supplier_account)
+    response = client.get('/suppliers/?location=tel+aviv&date=2023-01-18&add=Add')
+    assert len(DeliveryLocation.objects.filter(user_name=saved_supplier0)) != 0
+    assert response.status_code == 302
+    assert client.get('/suppliers/').status_code == 200
+
+
+@pytest.mark.django_db()
+def test_no_deliveries(client, saved_supplier0):
+    client.force_login(user=saved_supplier0.supplier_account)
+    response = client.get('/suppliers/')
+    assert len(DeliveryLocation.objects.filter(user_name=saved_supplier0)) == 0
+    assert response.status_code == 200
+    assert len(list(response.context['supplier_deliveries'])) == 0
+    assert response.context['message'] == "You don't have any deliveries yet"
+
+
+@pytest.mark.django_db()
+def test_delete_delivery(client, saved_supplier0):
```

Comment:
since the function is called `remove_delivery`, I would keep the same name on the test.